File: BonusServer/Models/NotifyData.cs
namespace BonusServer.Models
{
    public class NotifyData
    {
        public enum WinSpin
        {
            None,
            A,
            B,
            CR
        };

        public WinSpin WinSpinType { get; set; }
        public string WebSite { get; set; }
        public string Message {  get; set; }
        public int Scores { get; set; }
        public string MachineName { get; set; }
        public string UserAccount { get; set; }
        public DateTime CreateTime { get; set; }
        public int LifeInMinutes { get; set; }

        public NotifyData()
        {
            this.WinSpinType = WinSpin.None;
            this.WebSite = "";
            this.Message = "";
            this.Scores = 0;
            this.MachineName = "";
            this.UserAccount = "";
            this.CreateTime = DateTime.Now;
        }
    }
}

File: BonusServer_Node/models/notifyData.js


export default class NotifyData
{
    static WinSpin = {
        None: 0,
        A: 1,
        B: 2,
        CR: 3
    };

    WinSpinType;
    WebSite;
    Message;
    Scores;
    MachineName;
    UserAccount;
    CreateTime;
    LifeInMinutes;

    constructor()
    {
        this.WinSpinType = NotifyData.WinSpin.None;
        this.WebSite = "";
        this.Message = "";
        this.Scores = 0;
        this.MachineName = "";
        this.UserAccount = "";
        this.CreateTime = new Date();
    }
}